src/Core/Plugin.php

```diff
@@ -129,4 +129,14 @@ public static function getCollection(): PluginCollection
     {
         return static::$plugins ??= new PluginCollection();
     }
+
+    /**
+     * Reset the state which holds the loaded plugins
+     *
+     * @return void
+     */
+    public static function resetCollection(): void
+    {
+        static::$plugins = null;
+    }
```

Comment:
Without this method I wasn't able to test the different states of loaded plugins inside our Testsuite.